src/hooks/appDispatch.ts

```diff
@@ -0,0 +1,4 @@
+import { useDispatch } from 'react-redux';
+import type { AppDispatch } from '../store/store';
```

Comment:
We don't need to use "type" when we import types. Also we should name files in kebab case, so rename the file to use-app-dispatch.ts
```suggestion
import { useDispatch } from 'react-redux';

import { AppDispatch } from 'src/store';
```